src/ssmDocument/commands/publishDocument.ts

```diff
@@ -182,23 +185,55 @@ export async function updateDocument(
         outputChannel.appendLine(
             localize(
                 'AWS.message.info.ssmDocument.publishDocument.updateSuccess',
-                "Successfully updated SSM Document '{0}'",
+                "Successfully updated Systems Manager Document '{0}'",
                 wizardResponse.name
             )
         )
         if (result.DocumentDescription) {
             outputChannel.appendLine(stringify(result.DocumentDescription))
         }
-        logger.info(`Updated SSM Document successfully ${stringify(result.DocumentDescription)}`)
-        vscode.window.showInformationMessage(`Updated SSM Document successfully`)
+        logger.info(`Updated Systems Manager Document successfully ${stringify(result.DocumentDescription)}`)
+        vscode.window.showInformationMessage(`Updated Systems Manager Document successfully`)
         outputChannel.appendLine('')
+
+        const isConfirmed = await showConfirmationMessage(
+            {
+                prompt: localize(
+                    'AWS.ssmDocument.publishDocument.updateVersion.prompt',
+                    'Would you like to make this the default version for {0}?',
+                    wizardResponse.name
+                ),
+                confirm: localize('AWS.ssmDocument.publishDocument.updateVersion.confirm', 'Yes'),
+                cancel: localize('AWS.ssmDocument.publishDocument.updateVersion.cancel', 'No'),
+            },
+            window
+        )
+
+        if (!isConfirmed) {
+            logger.info('Declined update default version on update document success.')
+        } else {
+            try {
+                let documentVersion: string | undefined = result.DocumentDescription?.DocumentVersion
+                if (documentVersion !== undefined) {
+                    await client.updateDocumentVersion(wizardResponse.name, documentVersion)
+                    vscode.window.showInformationMessage(
+                        `Updated Systems Manager Document default version successfully`
+                    )
+                }
+            } catch (err) {
+                logger.info(
+                    `Failed to update Systems Manager Document '${wizardResponse.name}' default version. %0`,
+                    err as Error
+                )
+            }
```

Comment:
Is this try/catch redundant? Is the change in wording from this catch block important enough to necessitate this? Does there need to be any user feedback that the error was caught?